api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -3163,6 +3163,76 @@ public CohortDefinition getB2_13(boolean e53) {
+   *       Resumo com resultado > 50 cópias durante o período de inclusão (“Data da CV >50” >= “Data
+   *       Início Inclusão” e <= “Data Fim Inclusão”.
+   *   <li>Nota: em caso de existência de mais de um registo de Carga Viral com resultado > =10050
+   *       cópias, deve ser considerado o primeiro registo ocorrido durante o período de inclusão
+   * </ul>
+   *
+   * @return CohortDefinition <strong>Should</strong> <strong>Should</strong> Returns empty if there
+   *     is no patient who meets the conditions <strong>Should</strong> fetch all patients with B2
+   *     criteria
+   */
+  public CohortDefinition getB3_13(boolean e53) {
+
+    SqlCohortDefinition sqlCohortDefinition = new SqlCohortDefinition();
+    sqlCohortDefinition.setName(
+        "All patients from Ficha Clinica with “Carga Viral” registered with numeric value > 1000 during the Inclusion period");
+    sqlCohortDefinition.addParameter(new Parameter("startDate", "startDate", Date.class));
+    sqlCohortDefinition.addParameter(new Parameter("endDate", "endDate", Date.class));
+    sqlCohortDefinition.addParameter(new Parameter("location", "location", Location.class));
+    sqlCohortDefinition.addParameter(
+        new Parameter("revisionEndDate", "revisionEndDate", Date.class));
+
+    Map<String, Integer> map = new HashMap<>();
+    map.put("6", hivMetadata.getAdultoSeguimentoEncounterType().getEncounterTypeId());
+    map.put("53", hivMetadata.getMasterCardEncounterType().getEncounterTypeId());
+    map.put("856", hivMetadata.getHivViralLoadConcept().getConceptId());
+
+    String query =
+        " SELECT"
+            + "             p.patient_id"
+            + "             FROM"
+            + "             patient p"
+            + "                INNER JOIN"
+            + "             (SELECT"
+            + "                p.patient_id, MIN(e.encounter_datetime) AS first_encounter"
+            + "             FROM"
+            + "                patient p"
+            + "             INNER JOIN encounter e ON e.patient_id = p.patient_id"
+            + "             JOIN obs o ON o.encounter_id = e.encounter_id"
+            + "             WHERE p.voided = 0"
+            + "                    AND e.voided = 0"
+            + "                    AND o.voided = 0"
+            + "                    AND e.location_id = :location"
+            + "                    AND o.location_id = :location"
+            + "                    AND o.concept_id = ${856}"
+            + "                    AND o.value_numeric > 50"
+            + "                    AND ("
+            + "                         ( e.encounter_type = ${6} "
+            + "                         AND e.encounter_datetime BETWEEN :startDate AND :endDate) ";
+
+    if (e53) {
+      query +=
+          "                         OR (e.encounter_type = ${53} "
+              + "                         AND o.obs_datetime BETWEEN :startDate AND :endDate)";
+    }
```

Comment:
This method is only being used for this case, I think there no need to have a flag to filter unused cases.
I suggest removing this verification.

Reply:
Done